include/cglm/vec4.h

```diff
@@ -256,6 +258,44 @@ glm_vec4_norm(vec4 v) {
 #endif
 }
 
+/*!
+ * @brief L1 norm of vec4
+ *
+ * @param[in] v vector
+ *
+ * @return L1 norm
+ */
+CGLM_INLINE
+float
+glm_vec4_norm_one(vec4 v) {
+#if defined(CGLM_SIMD)
+  return glmm_norm_one(glmm_load(v));
+#else
+  vec4 t;
+  glm_vec4_abs(v, t);
+  return glm_vec4_hadd(t);
+#endif
+}
+
+/*!
+ * @brief infinity norm of vec4
+ *
+ * @param[in] v vector
+ *
+ * @return infinity norm
+ */
+CGLM_INLINE
+float
+glm_vec4_norm_inf(vec4 v) {
+#if defined(CGLM_SIMD)
```

Comment:
`CGLM_SIMD` is defined when one of SIMD is supported. If NEON_FP is present then this will (or should) be defined. But it seems `glmm_norm_inf()` is only defined for x86. In this case it you should test SIMD against `CGLM_SIMD_x86` not `CGLM_SIMD`, otherwise it cannot be compiled on ARM I guess.

Reply:
Yes, you are absolutely right. I will try to implement `glmm_norm_inf()` for ARM and keep the `CGLM_SIMD` definition. Otherwise I will switch to `CGLM_SIMD_x86`, as you said.